frontend/app/components/ui/modal/index.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class UiModalComponent extends Component {
+  @action
+  closeModal() {
+    this.args.onCloseModal();
+  }
+
+  @action
+  preventModalClose(evt) {
```

Comment:
where is this used

Reply:
it needs to be removed, earlier it is used to prevent closing the modal. changed the logic after that and forgot to remove